docs/content/aws.md

```diff
@@ -0,0 +1,391 @@
+# Running El Carro Operator on Amazon cloud (AWS)
+
+This guide helps you run El Carro Operator in [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/).
+
+If you prefer to use GKE (Google Kubernetes Engine) to deploy the El
+Carro Operator, stop here and refer to the [Quickstart Guide](quickstart.md) or
+[Quickstart Guide for Oracle 18c XE](quickstart-18c-xe.md).
+
+If you prefer to use a local cluster on your personal computer instead of AWS EKS, refer to either the [Minikube Guide](minikube.md) or [Kind Guide](kind.md).
```

Comment:
```suggestion
If you prefer to use a local cluster on your personal computer instead of a managed Kubernetes services such as AWS EKS or GGP GKE, refer to either the [Minikube Guide](minikube.md) or [Kind Guide](kind.md).
```